test/e2e_test.sh

```diff
@@ -79,6 +79,15 @@ docker run -d -p 5000:5000 --restart always -e REGISTRY_STORAGE_DELETE_ENABLED=t
 export COSIGN_TEST_REPO=localhost:5000
 go test -tags=e2e -v ./test/... -run TestSignVerifyClean
 
+# Test pkcs11 token signing
```

Comment:
Can we put this in its own shell script and workflow, following the pattern of other e2e tests, so it runs in parallel?